server/driver/clock_offset.cpp

```diff
@@ -146,3 +151,8 @@ XrTime clock_offset::to_headset(XrTime timestamp_ns) const
 {
 	return a * timestamp_ns + b;
 }
+
+bool clock_offset::is_stable() const
+{
+	return stable;
+}
```

Comment:
We already have an `operator bool()`.
We can change `clock_offset.cpp` to check for consistency before making it valid.